docs/site/content/docs/edge/getting-started-docker.md

```diff
@@ -0,0 +1,136 @@
+# Getting Started with Docker Desktop
+
+<!-- markdownlint-disable MD036 -->
+<!-- markdownlint-disable MD024 -->
+
+Docker Desktop offers a Tanzu Community Edition extension. This extension spins
+up a local cluster that you can interact with via `kubectl` and the `tanzu` CLI.
```

Comment:
Docker Desktop Extension users might not know what `unmanaged-clusters` are. From the extensions standpoint, they are just deploying a Tanzu kubernetes cluster through the docker dashboard. Further, many of the unmanaged-cluster operations (like, `list`, `start`, `stop` through the command line) are not available in the docker desktop extension.

But I agree, I think it may be useful to link to unmanaged-cluster so that if users want to dive in deeper, they can.